src/style-spec/reference/v8.json

```
@@ -3127,7 +3169,7 @@
         }
       },
       "properties": {
-        "doc": "Gets the feature properties object.  Note that in some cases, it may be more efficient to use [\"get\", \"property_name\"] directly.",
+        "doc": "Returns the feature properties object.  Note that in some cases, it may be more efficient to use [\"get\", \"property_name\"] directly.",
```

Do we want code formatting, for consistency? Only a tiny nit, but just to make sure downstream docs are rendered correctly:

```suggestion
        "doc": "Returns the feature properties object.  Note that in some cases, it may be more efficient to use `[\"get\", \"property_name\"]` directly.",
```